backend/src/services/payments/payment-checkout.service.ts

```
@@ -67,75 +63,60 @@ export class PaymentCheckoutService {
-          input.customerEmail ?? null,
-          JSON.stringify(input.lineItems),
-          input.successUrl,
-          input.cancelUrl,
-          input.idempotencyKey ?? null,
-          JSON.stringify(metadata),
-        ]
-      );
-      inserted = result.rowCount !== 0;
-      if (!inserted) {
-        existingCheckoutSession = await this.findMatchingIdempotentCheckoutSession(
+      return await withUserContext(this.getPool(), this.getSafeUserContext(user), async (client) => {
```

ESLint

[error] Replace `this.getPool(),·this.getSafeUserContext(user),` with `⏎········this.getPool(),⏎········this.getSafeUserContext(user),⏎·······`

(prettier/prettier)